main.go

```diff
@@ -446,20 +482,23 @@ func processLogFile(file string, enc *json.Encoder) (int, error) {
 
 	status := sensu.CheckStateOK
 	results := analyzer.Go(context.Background())
-
+	numResults := 0
 	for result := range results {
 		if result.Err != nil {
 			status = sensu.CheckStateCritical
 		}
 		if err := enc.Encode(result); err != nil {
-			return sensu.CheckStateCritical, fmt.Errorf("error couldn't encode result %+v for file %s: %s", result, result.Path, err)
-		}
-		if status < plugin.MatchStatus {
-			status = plugin.MatchStatus
+			return 0, fmt.Errorf("error couldn't encode result %+v for file %s: %s", result, result.Path, err)
 		}
+		numResults++
+		/*
+			if status < plugin.MatchStatus {
+				status = plugin.MatchStatus
+			}
+		*/
```

Comment:
Please remove or incorporate, not sure if this is needed?

Reply:
oops that was old logic i commented out while putting in the new logic... i'll remove